src/components/layouts/Page/index.js

```diff
@@ -41,8 +40,8 @@ const renderPageContent = ({
   isHome,
   onBackButtonClick,
 }) => (
-  <div css={isDesktop && styles.rightContainerDesktop}>
-    <div css={styles.content}>
+  <div css={[styles.container, isDesktop && styles.rightContainerDesktop]}>
+    <div css={[styles.content, isDesktop && styles.desktopContent]}>
```

Comment:
Can we just use are responsive styles to handle this change rather? 
```
content: css({
  height: ['100%', '', 'auto'],
})
```

Reply:
derp sorry 🤦‍♀